spec/index.html

```diff
@@ -363,6 +363,35 @@ <h3>Triple Terms and Reification</h3>
 
     <p>Note that a <a>triple term</a> may also have another <a>triple term</a> as an <a>object</a>.</p>
   </section>
+  
+  <section id="section-version-announcement">
+    <h3>RDF version announcement</h3>
```

Comment:
To be consistent, should be in "Title Case".

```suggestion
    <h3>RDF Version Announcement</h3>
```